apis/types/error.go

```diff
@@ -20,24 +20,24 @@ type Error struct {
 }
 
 // Validate validates this error
```

Comment:
Your commit message has exceeded one line，please use one statement to describe you PR. And you can write more contents in commit content. Besides, your commit message looks like no sense：

```
refactor: api/types/*.go code review, modify functions with return values
```
1. What is code review ?